### PR TITLE
Mkampen/add cmems

### PR DESCRIPTION
**Pull request by Kampen1988**

### CODE REVIEW

**Review by dthiex — APPROVED** [review]
Changes look good. Important to notice though is that the zarrs currently in use are not taken from here but are mocked data within `edc-browser\src\services\EDCHandler.js`.

In order for the EDC Browser to use the files directly you need to:
- extend the yaml with at least cube:dimensions and some other mandatory metadata so that the yaml to json generator generates stac jsons (for reference you can take a look at the mooked data within the path shared above
- check with Pontus if there would be any changes needed to the EDC Browser (I am not sure if zarrs from the json endpoint will directly work out of the box (see also discussion [here](https://git.sinergise.com/sh-vas/web-apps/edc-browser/-/issues/131#note_291914)

File: collections/oceancolour-atl-chl-l4-nrt-observations-009-037.yaml [hunk 1]
```diff
@@ -49,6 +51,41 @@ Resources:
    StoreTitle: Datasets from Copernicus Services
    DataID: OCEANCOLOUR_ATL_CHL_L4_NRT_OBSERVATIONS_009_037.zarr
    Notes: "[Open in CMEMS Viewer](https://view.marine.copernicus.eu/ViewService/?record_id=6783756a-2cf7-4af7-899b-a03f115940d4)"
-  
+Extent:
```
Comment (dthiex) [inline collections/oceancolour-atl-chl-l4-nrt-observations-009-037.yaml:54]: After adding this the json generator will generate stac jsons but the EDC Browser doesn't know yet how to display those. You need to add at least 1 evalscript. Something like:

```
Configurations:
  - layer_name: "True Color"
    evalscript_url: https://custom-scripts.sentinel-hub.com/sentinel2-120m-mosaic/true-color/script.js
    mosaicking_order: mostRecent
    upsampling: BICUBIC
    href: https://docs.sentinel-hub.com/api/latest/api/process/
    rel: processing-expression
```

Comment (Kampen1988) [reply collections/oceancolour-atl-chl-l4-nrt-observations-009-037.yaml:54]: Not sure how to add this information for collections managed by Brockmann Consult. I feel like I can only edit the entries up to this point.

Comment (dthiex) [reply collections/oceancolour-atl-chl-l4-nrt-observations-009-037.yaml:54]: Oh, I didn't see that the collection is managed by Brockmann and not a BYOC collection. I think so far there was/is no support in EDC Browser for Brockmann's raster collection. We'd need to check how those can be accessed and then 
- update the metadata with the required info
- change the EDC Browser code to read those correctly

**Review by dthiex — APPROVED** [review]
Looks good!

**Pull request merged**